pint/utils.py

```diff
@@ -439,6 +440,74 @@ def is_number(s):
         pass
     return False
 
+def make_toas(startMJD, endMJD, ntoas, model, freq=1400, obs='GBT'):
+    '''make evenly spaced toas without errors'''
+    def get_freq_array(bfv, ntoas):
+        freq = np.zeros(ntoas)
+        num_freqs = len(bfv)
+        for ii, fv in enumerate(bfv):
+            freq[ii::num_freqs] = fv
+        return freq
+    
+    times = np.linspace(np.longdouble(startMJD)*u.d, np.longdouble(endMJD)*u.d, ntoas) * u.day
+    freq_array = get_freq_array(np.atleast_1d(freq)*u.MHz, len(times))
+    t1 = [pint.toa.TOA(t.value, obs = obs, freq=f, \
```

Comment:
No need for a backslash when you have open brackets. Also indentation should line up with the previous line's appropriate part, here scale= should be directly below t.value. See PEP8.

Reply:
Fixed